lib/Completion/Bridge/TolerantParser/WorseReflection/KeywordCompletor.php

```diff
@@ -58,6 +76,19 @@ public function complete(Node $node, TextDocument $source, ByteOffset $offset):
         return true;
     }
 
+    /**
+     * @return Generator<Suggestion>
+     */
+    private function methods(): Generator
+    {
+        foreach (self::MAGIC_METHODS as $name => $snippet) {
+            yield Suggestion::createWithOptions($name . '(', [
+                'type' => Suggestion::TYPE_METHOD,
+                'priority' => 1,
+            ] + ['snippet' => $name . $snippet]);
```

Comment:
why don't we make `snippet` part of the array? (i.e. why use `+`?)

Reply:
A habit from other places where it was conditional...